GitExtUtils/Delimiters.cs

```diff
@@ -0,0 +1,22 @@
+using System.IO;
+
+namespace System
+{
+    internal static class Delimiters
+    {
+        public static readonly char[] Newline = { '\n' };
+        public static readonly char[] Space = { ' ' };
+        public static readonly char[] Tab = { '\t' };
+        public static readonly char[] Null = { '\0' };
+        public static readonly char[] TabAndSpace = { '\t', ' ' };
+        public static readonly char[] TabAndNewlineAndCarriageReturn = { '\t', '\n', '\r' };
+        public static readonly char[] NullAndNewline = { '\0', '\n' };
+        public static readonly char[] NewlineAndCarriageReturn = { '\n', '\r' };
```

Comment:
```suggestion
        public static readonly char[] CrLf = { '\n', '\r' };
```

Reply:
I feel like "new line" is more commonly understood than "line feed", although "line feed" is more specific.

Reply:
"New line" means different thing on different platforms.